app/api/getBase64/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse } from "next/server"
```

Comment:
What is you idea of using the `api` in the route? 
How is it going to look like when we are introducing the Publizon API?